monkey/infection_monkey/telemetry/aws_instance_telem.py

```diff
@@ -0,0 +1,19 @@
+from common.common_consts.telem_categories import TelemCategoryEnum
+from infection_monkey.telemetry.base_telem import BaseTelem
+
+
+class AwsInstanceTelemetry(BaseTelem):
+    def __init__(self, aws_instance_info):
```

Comment:
no type hint for `aws_instance_info`. Perhaps this should just accept resource ID